andela/travel_tool_back
# #162039447: Fix seed data for Nairobi guesthouses in Nairobi

#### What does this PR do?
Fix seed data for Nairobi guesthouse.

#### Description of Task to be completed?
- Change seed data for Nairobi guest house location from `Nairobi` to `Nairobi, Kenya` to show available rooms on the front end when creating a request to `Travel To` Nairobi.

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Enter into the project directory - `cd travel_tool_back`
- Switch to the `bg-fix-guesthous-seed-data-162039447` branch.
- Rollback, run database migrations and seed database - `yarn db:rollmigrate`
- Install dependencies and start the server - `make start`
- Then cd out of the folder and `git clone https://github.com/andela/travel_tool_back.git`
- Install dependencies and start the server - `make start`
- Login and navigate to the `Requests`, then create a `New Request` and set `Travel to` to Nairobi, Kenya.
- The available beds should read `Nyati Bed 1` and any other available beds in Nairobi.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162039447](https://www.pivotaltracker.com/story/show/162039447)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-11-19 at 19 41 49" src="https://user-images.githubusercontent.com/12511966/48721557-361a9880-ec33-11e8-99cb-fe7cb4a0a681.png">


#### Questions:
N/A
